opdracht-1.js

```diff
@@ -0,0 +1,27 @@
+//Opdracht 1a
+const tvTypes = inventory.map((tvType) => {
+    return tvType.type;
+});
+
+console.log(tvTypes)
+
+
+//opdracht 1b
+const soldOutTvs = inventory.filter((soldOutTv)=> {
+return soldOutTv.sold === soldOutTv.originalStock
+});
+
+console.log(soldOutTvs)
+
+//Opdracht 1c
+const ambiLightTvs = inventory.filter((ambiLighttv) => {
+    return ambiLighttv.options.ambiLight === true
```

Comment:
`boolean_waarde === true` is altijd gelijk aan `boolean_waarde`, dus in dit geval
```suggestion
    return ambiLighttv.options.ambiLight
```